api/models/Comment.js

```diff
@@ -39,12 +40,33 @@ module.exports = bookshelf.Model.extend({
     return this.hasMany(Activity)
   },
 
+  comment: function () {
+    return this.belongsTo(Comment)
+  },
+
+  comments: function () {
+    return this.hasMany(Comment, 'comment_id').query({where: {active: true}})
+  },
+
+  followers: function () {
+    return this.belongsToMany(User).through(Follow).withPivot('added_by_id')
+  },
+
   createActivities: function (trx) {
-    return this.load(['post', 'post.followers'])
+    const isReplyToPost = !this.get('comment_id')
+
+    var toLoad = ['post', 'post.followers']
+    if (!isReplyToPost) toLoad = toLoad.concat(['comment', 'comment.followers'])
+
+    return this.load(toLoad)
     .then(() => {
-      const followers = this.relations.post.relations.followers.map(follower => ({
+      const followers = (isReplyToPost
```

Comment:
`this.relations[isReplyToPost ? 'post' : 'comment'].relations.followers` is a bit more concise